app/views/tags/edit.js.erb

```diff
@@ -0,0 +1,2 @@
+$('[data-behavior="modal-tag-form"]').modal('show');
+$('[data-behavior="modal-tag-content"]').html("<%= j (render partial: 'form', locals: { header: 'Edit tag', url: project_tag_path(current_project, @tag),  method: :patch }) %>");
```

Comment:
```suggestion
$('[data-behavior~=modal-tag-form]').modal('show');
$('[data-behavior~=modal-tag-content]').html("<%= j (render partial: 'form', locals: { header: 'Edit tag', url: project_tag_path(current_project, @tag),  method: :patch }) %>");
```

We like to use `~=` rather than `==` because we can stack additional behaviors in the view but only use the ones we need in JS. For example, a view could have `data-behavior="123 ABC"` but in the JS that only cares about `ABC` we can use `$('[data-behavior~=ABC]')`.